neptune/alpha/experiment.py

```diff
@@ -61,10 +64,24 @@ def __init__(
         self._bg_job = background_job
         self._structure = ExperimentStructure[Attribute]()
         self._lock = threading.RLock()
+        self._started = False
 
     def start(self):
+        atexit.register(self._shutdown_hook)
         self._op_processor.start()
         self._bg_job.start(self)
+        self._started = True
+
+    def stop(self, seconds: Optional[float] = None):
+        if not self._started:
+            return
+        self._started = False
+        ts = time.time()
+        self._bg_job.stop()
+        self._bg_job.join(seconds)
+        with self._lock:
+            sec_left = None if not seconds else seconds - (time.time() - ts)
```

Comment:
i don't know what is further expected semantic, but it would be more consistent to use if seconds is None instead of not seconds.

Reply:
Done.